2.6/root/usr/share/container-scripts/mongodb/replset-functions.sh

```diff
@@ -0,0 +1,91 @@
+# cache_container_addr waits till the container gets the external IP address and
+# cache it to disk
+function cache_container_addr() {
+  echo -n "=> Waiting for container IP address ..."
+  for i in $(seq $MAX_ATTEMPTS); do
+    result=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')
```

Comment:
@omron93 this function also changed recently, could you please update?

Here is the new code:
https://github.com/openshift/mongodb/blob/master/2.4/root/usr/share/container-scripts/mongodb/common.sh#L26-L39